pyregion/tests/test_region.py

```diff
@@ -1,47 +1,69 @@
 import os
-import numpy as np
 from os.path import join
 from astropy.io.fits import Header
-from ..wcs_helper import fix_lon
 from .. import open as pyregion_open
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
 
-rootdir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'examples')
+rootdir = join(os.path.dirname(os.path.abspath(__file__)), 'data')
```

Comment:
I don't think that would be better. `rootdir = get_pkg_data_filename("data")` [raises an error](https://github.com/astropy/astropy/blob/master/astropy/utils/data.py#L594). If I use that function for each file, then `join(rootdir, header_name)` grows to the verbose `get_pkg_data_filename(join("data", header_name))`